src/openforms/registrations/contrib/objects_api/utils.py

```diff
@@ -0,0 +1,10 @@
+from .models import ObjectsAPIConfig
+
+
+def get_registration_json_templates(submission) -> dict:
+    config = ObjectsAPIConfig.get_solo()
+    form_options = submission.form.registration_backend_options
```

Comment:
Maybe good to only retrieve the global if there is no template specified on the form? Then you would save a query.

Also, in [src/openforms/registrations/contrib/objects_api/models.py](https://github.com/open-formulieren/open-forms/pull/3160/files#diff-554ceb9c5db4b300f57e6a76cc6247c1482f0052f904f8ceae12e44d0cddb2c2) you set the default on the form in case it's not specified. So if you keep that, maybe this is not needed?